proto/requests.proto

```diff
@@ -135,6 +135,9 @@ message GetAccountStateDeltaRequest {
 message GetAccountProofsRequest {
     // List of account IDs to get states.
     repeated account.AccountId account_ids = 1;
+    // Account code commitments corresponding to the last-known `AccountCode` for requested 
+    // accounts. Responses will include only the ones that are not known to the caller.
+    repeated digest.Digest code_commitments = 2;
     // Optional flag to include header in the response. `false` by default.
-    optional bool include_headers = 2;
+    optional bool include_headers = 3;
```

Comment:
nit: I would move `include_headers` above `code_commitments`.

Also, we should update comments to make it clear that account code only is included if `include_headers` is true, and also that it is not associated with a specific account ID (i.e., we check code roots against all accounts).